.gitignore

```diff
@@ -1 +1,3 @@
 /node_modules/
+/.idea/
+package-lock.json
```

Comment:
This suggest that package-lock.json files should be included in version control. We don't have a package-lock.json, but we might want to add it.
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5